code/game/jobs/jobs.dm

```diff
@@ -65,12 +65,19 @@ var/list/command_positions = list(
 	"Head of Security",
 	"Chief Engineer",
 	"Research Director",
-	"Chief Medical Officer"
+	"Chief Medical Officer",
+	"Acting Captain",
+	"Acting Head of Personnel",
+	"Acting Head of Security",
+	"Acting Chief Engineer",
+	"Acting Research Director",
+	"Acting Chief Medical Officer"

```

Comment:
I agree with @SamHPurp - on a crew Manifest there should be full titles used. We only use abbreviations to shorten having to type "Head of Personnel" for example in the chat repeatedly.